docs/index.rst

```diff
@@ -525,13 +526,19 @@ API
 .. autoclass:: BadHeaders
    :members:
 
+
+Change History
+--------------
+
+.. include:: ../CHANGES.rst
+
+
```

Comment:
Prefer pulling this in as a separate page to `.. include::`.

Reply:
Sphinx does not support `toctree` entries for files outside the docs root ([sphinx-doc/sphinx#701](https://github.com/sphinx-doc/sphinx/issues/701)) unless it is a fully qualified URL, so the include is necessary.

Reply:
You can create a symlink. `docs/changes.rst` -> `../CHANGES.rst`.

Reply:
Oh, cool! I never thought of that. It works. Fixed in 8b432260d629604a5db7160d12b66d1a508a1a47

Reply:
Please don't do this... anyone that checks out the repo on Windows is going to deal with a broken repo.

Reply:
@bertjwregeer Please see https://blogs.windows.com/buildingapps/2016/12/02/symlinks-windows-10/.  I've been using this technique without pain for $DAYJOB for a couple of years now.

Reply:
Last time I had to deal with Git on Windows was Windows 8.1, so it looks like it is finally fixed. Leave the symlink then.